dbt/include/dremio/macros/materializations/incremental/incremental.sql

```diff
@@ -14,64 +14,96 @@ limitations under the License.*/
 
 {% materialization incremental, adapter='dremio' -%}
 
-  {%- set identifier = model['alias'] -%}
-  {%- set old_relation = adapter.get_relation(database=database, schema=schema, identifier=identifier) -%}
-  {%- set target_relation = api.Relation.create(identifier=identifier,
-                                                schema=schema,
-                                                database=database,
-                                                type='table') -%}
-
-  {%- set tmp_identifier = model['name'] + '__dbt_tmp' -%}

```

Comment:
Is there a reason we're getting rid of the tmp_identifier?

Reply:
tmp_identifier was used in the old logic to get tmp_relation and create target_tmp_relation. The new logic has moved to temp_relation and intermediate_relation with a different relation creation strategy, therefore, it is no longer needed.